src/components/Parameters/Parameters.js

```diff
@@ -0,0 +1,35 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import BodySchema from '../BodySchema/BodySchema'
+
+export default class Parameters extends PureComponent {
+  render () {
+    const {parameters, classes} = this.props
+
+    if (!parameters) {
+      return null
+    }
+
+    const parameterUIElements = []
+    for (const key in parameters) {
```

Comment:
No reason. It was just a clumsy way. I've updated the code to use `Object.keys...`